test/index.js

```diff
@@ -262,9 +262,8 @@ describe('hyperlink', function () {
         expect(t.push, 'to have a call satisfying', () => {
             t.push(null, {
                 ok: false,
-                operator: 'content-type-missing',
-                name: 'content-type-missing https://example.com/hey.png',
-                expected: 'image/png',
+                operator: 'error',
```

Comment:
I don't like the generic `error` operator. Can't be properly filtered on. We should probably handle that error explicitly and trigger the right operator